filebeat/prospector/udp/harvester.go

```diff
@@ -4,11 +4,12 @@ import (
 	"net"
 	"time"
 
-	"github.com/elastic/beats/filebeat/harvester"
-	"github.com/elastic/beats/filebeat/util"
+	"github.com/elastic/beats/libbeat/beat"
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/logp"
-	"github.com/elastic/beats/libbeat/publisher/beat"
+
+	"github.com/elastic/beats/filebeat/harvester"
```

Comment:
Yes, but not with the standard `goimports`. It would take a customization to `goimports`, which isn't a bad idea, it would just take a little time.